python/dask_cudf/dask_cudf/backends.py

```diff
@@ -429,7 +429,7 @@ def union_categoricals_cudf(
 
 @_dask_cudf_nvtx_annotate
 def safe_hash(frame):
-    return cudf.Series(frame.hash_values(), index=frame.index)
+    return frame.hash_values()
```

Comment:
Nice - Sure looks like we don't need `safe_hash` at all anymore :)

Reply:
Dropping it 😉

Reply:
Dropped it